forwards_paper.tex

```diff
@@ -115,14 +119,15 @@ \section*{Introduction}
 in a population having even hundreds of millions of individuals.
 However, if our interest lies in the resulting genetic patterns of variation
 -- and often, the point of such simulations is to compare to real data --
-then such simulations must somehow produce at the end data for each individual on a genomic scale.
+then such simulations must record data for each individual genome.
```

Comment:
wanted to emphasize they need to create *whole genome* data.  how about "must record each individual's genome." ?

Reply:
to address comment below, how about "must record neutral variants at the large number of locations in each individual's genome"